titiler/pgstac/factory.py

```diff
@@ -67,6 +69,17 @@ def _first_value(values: List[Any], default: Any = None):
     return next(filter(lambda x: x is not None, values), default)
 
 
+DEFAULT_TEMPLATES = Jinja2Templates(
+    directory="",
+    loader=jinja2.ChoiceLoader(
+        [
+            jinja2.PackageLoader(__package__, "templates"),
+            jinja2.PackageLoader("titiler.core", "templates"),
```

Comment:
We need this because we are using a modified version of the wmts.xml file from titiler